common/test/acceptance/tests/lms/test_conditional.py

```diff
@@ -0,0 +1,134 @@
+"""
+Bok choy acceptance tests for conditionals in the LMS
+
+See also old lettuce tests in lms/djangoapps/courseware/features/conditional.feature
+"""
+from capa.tests.response_xml_factory import StringResponseXMLFactory
+from ..helpers import UniqueCourseTest
+from ...fixtures.course import CourseFixture, XBlockFixtureDesc
+from ...pages.lms.courseware import CoursewarePage
+from ...pages.lms.conditional import ConditionalPage, POLL_ANSWER
+from ...pages.lms.problem import ProblemPage
+from ...pages.studio.auto_auth import AutoAuthPage
+
+
+class ConditionalTest(UniqueCourseTest):
+    """
+    Test the conditional module in the lms.
+    """
+
+    def setUp(self):
+        super(ConditionalTest, self).setUp()
+
+        self.username = 'username'
```

Comment:
If you don't need to set specific username/password info, just call AutoAuthPage(self.browser).visit(). It will automatically create unique user info for you.

You can get reference to what was created (if necessary) in this way:

````
auto_auth = AutoAuthPage(self.browser).visit()
user_info = auto_auth.user_info